IP-addr.cidr.list

```diff
@@ -1,2 +1,3 @@
 # example address
 # 192.0.2.1/24 # rfc:5737
+95.181.157.0/24
```

Comment:
Are you sure you will go with a /24??? that is all 255 ip addresses on a C network. that is a lot of hosts

Reply:
can you post that as a screenshot?

Reply:
PS in the title you have wrote the 32 CIDR = 1 ip

Reply:
Screenshot of Spider  I think you have misunderstood something of what you have seen.

I think you have been reading the network for which the IP belongs to as the net-group for which is should be blocked

It is rare spamhaus marks a cidr 24 bad